library/cwm/src/lib/cwm/replace_point.rb

```diff
@@ -20,7 +20,16 @@ def initialize(id: "_placeholder", widget: Empty.new("_initial_placeholder"))
 
     # @return [UITerm]
     def contents
-      ReplacePoint(Id(widget_id), @widget)
+      # Use for contents empty widget, because if @widget passed to constructor
+      # will be used then CWM itself will handle events for this initial widget.
+      # This is against replace point idea that replace point due to its dynamic
+      # content will handle events itself
+      ReplacePoint(Id(widget_id), Empty(Id("___cwm_rp_empty")))
```

Comment:
@mvidner ok, so let me explain it. if contents returns WidgetTerm that include Widget, then this widget will be part of cwm map, so CWM will forward to it its events. But when it is replaced, it have to be handled by ReplacePoint. It is something that was not designed from start in CWM, so it need a bit additional handling on top of it.

I can dry this empty stuff.